public/app/features/dashboard/dashgrid/PanelStateWrapper.tsx

```diff
@@ -653,56 +652,16 @@ export class PanelStateWrapper extends PureComponent<Props, State> {
       />
     );
 
-    const overrideStyles: { menuItemsClassName?: string; menuWrapperClassName?: string; pos?: React.CSSProperties } = {
-      menuItemsClassName: undefined,
-      menuWrapperClassName: undefined,
-      pos: { top: 0, left: '-156px' },
-    };
-
+    const dragClass = !(isViewing || isEditing) ? 'grid-drag-handle' : '';
     if (config.featureToggles.newPanelChromeUI) {
-      // set override styles
-      overrideStyles.menuItemsClassName = css`
-        width: inherit;
-        top: inherit;
-        left: inherit;
-        position: inherit;
-        float: inherit;
-      `;
-      overrideStyles.menuWrapperClassName = css`
-        position: inherit;
-        width: inherit;
-        top: inherit;
-        left: inherit;
-        float: inherit;
-        .dropdown-submenu > .dropdown-menu {
-          position: absolute;
-        }
-      `;
-      overrideStyles.pos = undefined;
-    }
+      // Shift the hover menu down if it's on the top row so it doesn't get clipped by topnav
+      const hoverHeaderOffset = (panel.gridPos?.y ?? 0) === 0 ? -16 : undefined;
 
-    // custom styles is neeeded to override legacy panel-menu styles and prevent menu from being cut off
-    let menu;
-    if (!dashboard.meta.publicDashboardAccessToken) {
```

Comment:
Do we not need to have this check anymore?
I put because it is currently how it works: https://github.com/grafana/grafana/blob/main/public/app/features/dashboard/dashgrid/PanelHeader/PanelHeader.tsx#L63

Reply:
it needs to be inside the if newPanelChromeUI, and logic should be if dashboard.meta.publicDashboardAccessToken then set menu to undefined (ie no menu)